index.html

```diff
@@ -2171,11 +2229,23 @@ <h5 id="Mortality Traits">Mortality Traits</h5>
                </td>
                <td>1 AP - No Use Limit
                </td>
-               <td>Nature Level 1 or Mortality level 1
+               <td>Nature Level 3 or Mortality level 3
                </td>
             </tr>
             <tr>
-               <td>Speak With Dead
+               <td>Detect Life
+               </td>
+               <td>
+                  You may sense living creatures in a 15ft radius as spots of soul energy when you close your eyes
+                  The larger the creature, the larger and brighter the the spots are.
```

Comment:
Kinda same but stronger than the new psionic one? This range is more acceptable, but all things have soul energy - even undead, it's just unstable. Maybe each of these could be more situational somehow? The psionic one has the INT requirement which helps, maybe this one has a SPI requirement? They both just need something to make them a fair bit less consistent.